save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -377,17 +377,26 @@ internal class OrganizationController(
             // Find and remove all corresponding data to the current git repository from DB and file system storage
             val git = gitService.getByOrganizationAndUrl(organization, url)
             val testSuitesSources = testSuitesSourceService.findByGit(git)
+            // List of test suites for removing data from storage at next step
+            val testSuitesList: MutableList<TestSuite?> = mutableListOf()
             testSuitesSources.forEach { testSuitesSource ->
```

Comment:
mutable list can be avoided by returning result from `forEach` (by changing to `map`)

Reply:
Yeah, changed